bedrock/firefox/firefox_details.py

```diff
@@ -143,6 +150,7 @@ def latest_builds(self, locale, channel='release'):
                 # Append 64-bit builds
                 if 'Windows' in _builds:
                     _builds['Windows 64-bit'] = _builds['Windows']
+                    # TODO do we need to do anything here for MSI?
```

Comment:
@pmac one area of code I wasn't sure about was here. I'm not entirely clear on the need for special casing 64-bit builds. But it made me wonder do we need to do anything for MSI, or is it safe to remove this todo?

Reply:
Oh right. I saw that and meant to address it after running locally but forgot. Surprisingly I don't think the `latest_builds` function is used for much of anything anymore. It's called in the `desktop_builds` function in `helpers.py`, but the data isn't used. Even the version returned that is passed to `get_download_url()` isn't used by the function anymore since it no longer produces any bouncer links with version numbers. For now you can safely ignore this, but soon we should be able to clean all of this up a bit more and remove some of this legacy code.